Fullstacker/Api/Controllers/ItemController.cs

```diff
@@ -48,6 +52,20 @@ public IActionResult Get(int id)
         [Route("create")]
         public IActionResult Create([FromBody] CreateItemDto request)
         {
+            if (request == null)
+            {
+                return BadRequest();
+            }
+
+            var newItem = new DataLayer.Entities.Item{ 
+
+                Name = request.Name ,
+                Description = request.Description
+            };
+
+            _dataContext.Add(newItem);
+            _dataContext.SaveChanges();
```

Comment:
Good. For bonus points, make both the `Add` and `SaveChanges` asynchronous. Hint: You'll need to make this Create Method asynchronous as well.